### ISSUE TITLE
Small issues with the EpicFight mod. 

### ISSUE THREAD

**Issue by nc5432** [Globox1997/RpgZ#20]
If you use this mod with the epic fight mod, death animations loop and the hitboxes are a little off to the side. It just makes getting the loot a little more difficult since you don't actually click on the body, and it looks a little strange.

**Comment by Stereowalker** [comment]
EpicFight mod is a Forge mod right?

**Comment by nc5432** [comment]
> EpicFight mod is a Forge mod right?

Yes, EpicFight is a forge mod. I am using the 0.4.5 version of RpgZ for 1.16.5 and the 16.5.13 version of EpicFight.

**Comment by Siscuth-Siignossum** [comment]
Same issue.
